conf/agent/agent_full.conf

```diff
@@ -14,6 +14,25 @@ agent {
     # join_token = ""
 
     # log_file: File to write logs to.
+    #
+    # If set, spire-agent will spawn a handler to reopen the file upon receipt
+    # of SIGUSR2 to support log rotation. To use logrotate without lossy
+    # copytruncate option, users MUST add a postrotate script to the logrotate
+    # configuration to send the SIGUSR2 signal to the spire-agent process.
+    #
+    # Example logrotate.conf:
+    #
+    # /path/to/spire-agent.log {
```

Comment:
I'm not familiar with the logrotate config, but is this the absolute minimal config we can put in the example while still showing the necessary 2 lines? Want to make sure the necessary usage is totally clear

Reply:
I suppose I could reduce it to:
```
/path/to/spire-agent.log {
	rotate 7
	postrotate
		killall -USR2 spire-agent
	endscript
}
```

Reply:
might want to include the `compress` and `delaycompress` as recommendations, but `rotate` and `postrotate` are the minimum config. to use with this.